openassessment/workflow/models.py

```diff
@@ -228,6 +228,12 @@ def status_details(self):
                 "complete": step.is_submitter_complete(),
                 "graded": step.is_assessment_complete(),
             }
+            if step.name == 'peer':
+                # the number passed here is arbitrary and ignored
+                _, peers_graded = step.api().has_finished_required_evaluating(self.submission_uuid, 1)
+                graded_by_count = step.api().get_graded_by_count(self.submission_uuid)
+                status_dict[step.name]['peers_graded'] = peers_graded
+                status_dict[step.name]['graded_by_count'] = graded_by_count
```

Comment:
Please update the comment for this method to describe the fields added here.

And update the unit tests for `status_details` to test the newly added fields.